### ISSUE TITLE
Issue #2 [Priority-High]

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/issue-export-test#2]
```
This is a new issue. Initially it has label "Priority-Low".
```

-----
Original issue reported on code.google.com by chrsm...@google.com on 7 Jan 2015 at 5:57
* Blocked on: #7

**Comment by GoogleCodeExporter** [comment]
```
This is a comment, disregard.
```

-----
Original comment by chrsm...@google.com on 7 Jan 2015 at 5:58

**Comment by GoogleCodeExporter** [comment]
```
This comment sets the Priority label to High.
```

-----
Original comment by chrsm...@google.com on 7 Jan 2015 at 5:58
* Added labels: **Priority-High**
* Removed labels: **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by chrsm...@google.com on 7 Jan 2015 at 6:02
* Now blocked on: #7

**Comment by GoogleCodeExporter** [comment]
```
This issue is now blocked on issue #7.
```

-----
Original comment by chrsm...@google.com on 7 Jan 2015 at 6:03
